### PR TITLE
July 14 support provisions

### PR DESCRIPTION
**Pull request by GaryRidgway**
# How to test

### CODE REVIEW

File: docroot/sites/bnelson.lab.uiowa.edu/blt.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    local: self
+    remote: labbnelson.prod
+drupal:
+  db:
+    database: bnelson_lab_uiowa_edu
+uiowa:
+  stage_file_proxy:
+    origin: 'https://labbnelson.prod.drupal.uiowa.edu'
+  requester: bnelson3
+  site-name: bnelson3
```
Comment (joewhitsitt) [inline docroot/sites/bnelson.lab.uiowa.edu/blt.yml:18]: did the provision script mess this up or was this actually the value for site name?

**Review by cory-skeers — APPROVED** [review]
I kind of wish they'd asked for choirauditions.org.uiowa.edu instead of uiowachoirauditions.org.uiowa.edu, but I don't think the full one is a problem outside of being long with an extra uiowa.

File: docroot/sites/bnelson.lab.uiowa.edu/blt.yml (hunk 1 above)
Comment (cory-skeers) [reply docroot/sites/bnelson.lab.uiowa.edu/blt.yml:18]: It's in the request form as that. Not a good site name to launch with, but the provision script functioned as expected with it.

Comment (GaryRidgway) [reply docroot/sites/bnelson.lab.uiowa.edu/blt.yml:18]: i think that is the correct value, as the hawkid is `bnelson`

**Comment by joewhitsitt** [comment]
wait, we are okay with the double iowa?

**Comment by GaryRidgway** [comment]
I'm pinging Michael now, Hostmaster didn't flag it, but I can undo if necessary before release

**Comment by bspeare** [comment]
It's our job to flag the double uiowa :)

**Comment by GaryRidgway** [comment]
MJ said it is good on an org site, so we are good to go

**Pull request merged**